README.rst

```diff
@@ -305,6 +305,14 @@ This is the list of operators that can be used:
 - ``not_ilike``
 - ``in``
 - ``not_in``
+- ``any``
+- ``not_any``
+
+any / not_any
+^^^^^^^^^^^^^
+
+PostgreSQL specific operators allows to filter queries on ARRAY type columns.
```

Comment:
We could rephrase this description like this, to also highlight ``ARRAY`` as code:

```
PostgreSQL specific operators allow to filter queries on columns of type ``ARRAY``.
Use ``any`` to filter if a value is present in an array and ``not_any`` if it's not.
```

Reply:
done